events/roommessageevent.cpp

```diff
@@ -70,6 +70,18 @@ MessageEventContent::Base* RoomMessageEvent::content() const
     return d->content;
 }
 
+template <class ContentT>
+MessageEventContent::Base* make(const QJsonObject& json)
+{
+    return new ContentT(json);
+}
+
+template <class ContentT>
+MessageEventContent::Base* make2(const QJsonObject& json)
```

Comment:
A minor nitpicking issue: I don't really like the name "make2", as we might end up with make1-make10. Unfortunately, I don't really have a good name myself... Maybe just use make for VideoContent and AudioContent? VideoContent might use ThumbnailContent in the future, too, as it is currently an inconsistency